### PR TITLE
KYLIN-3766 Add docs for Query SQL Grammar, Function and Data Type

### PR DESCRIPTION
**Pull request by GinaZhai**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: website/_docs/tutorial/kylin_datatype.cn.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+categories: tutorial
+permalink: /cn/docs/tutorial/kylin_datatype.html
+---
+
+## 数据类型
+
+| :--------- | :--------- | :--------- | :--------- | :------------------- |
+| ANY        | CHAR       | VARCHAR    | STRING     | BOOLEAN              |
+| BYTE       | BINARY     | INT        | SHORT      | LONG                 |
+| INTEGER    | TINYINT    | SMALLINT   | BIGINT     | INT4 (for test only) |
```
Comment (shaofengshi) [inline website/_docs/tutorial/kylin_datatype.cn.md]: What's the "for test only"? If it is not recommended for end user, just remove it here.

File: website/_docs/tutorial/kylin_function.cn.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+
+## COUNT_DISTINCT {#COUNT_DISTINCT}
+
+例子：
+{% highlight Groff markup %}
+SELECT COUNT(DISTINCT seller_id) AS DIST_SELLER FROM kylin_sales;
+{% endhighlight %}
+
+## EXTENDED_COLUMN {#EXTENDED_COLUMN}
```
Comment (shaofengshi) [inline website/_docs/tutorial/kylin_function.cn.md]: This is not a function, we can remove it.

File: website/_docs/tutorial/kylin_grammar.cn.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+## 语法
+
+[SELECT](#SELECT)
+　[STATEMENT](#STATEMENT)
+　[EXPRESSION](#EXPRESSION)
+[SUBQUERY](#SUBQUERY)
+[JOIN](#JOIN)
+　[INNER JOIN](#INNERJOIN)
+　[LEFT JOIN](#LEFTJOIN)
+　[CROSS JOIN](#CROSSJOIN)
```
Comment (shaofengshi) [inline website/_docs/tutorial/kylin_grammar.cn.md]: As I know Kylin doesn't support "cross join", is it?

File: website/_docs/tutorial/kylin_function.cn.md (hunk 2 above)
Comment (GinaZhai) [reply website/_docs/tutorial/kylin_function.cn.md]: ok

**Pull request merged**